tests/torchtune/models/phi3/test_phi3_tokenizer.py

```diff
@@ -0,0 +1,252 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from pathlib import Path
+
+import pytest
+from torchtune.data import Message
+from torchtune.models.phi3 import phi3_mini_tokenizer
+
+ASSETS = Path(__file__).parent.parent.parent.parent / "assets"
+
+
+class TestPhi3MiniTokenizer:
```

Comment:
at least phi3 should be tested separately since the tokenize_messages is customized, but the others that simply use `tokenize_messages_no_special_tokens` there's not much difference. but it will be updated in the near future once they have their own special tokens (I need to look into Gemma more at least)